packs/classfeatures/physical-protections.json

```
@@ -24,7 +24,21 @@
             "remaster": false,
             "title": "Pathfinder Guns & Gears"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "Resistance",
+                "predicate": [
+                    "armor:id:{actor|flags.pf2e.innovationId}"
+                ],
+                "type": [
+                    "slashing",
+                    "piercing",
+                    "bludgeoning",
+                    "bleed"
+                ],
```

```suggestion
                "type": [
                    "physical"
                ],
```
Don't know why Paizo spelled it out, all of them are just different types of physical damage.